src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form_big"
+      action="search"
+      data-qa="big"
+    >
+      <input
+        class="input form_big__input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form_small"
+      action="search"
```

Comment:
Don't use spaces in links. The `action` attribute of the form should be a valid URL. If you're using a placeholder for the action, consider something like '/search'.